src/skmultiflow/data/agrawal_generator.py

```diff
@@ -266,39 +312,38 @@ def generate_drift(self):
             new_function = self.random_state.randint(10)
         self.classification_function_idx = new_function
 
-
     @staticmethod
-    def classification_function_zero(salary, commission, age, elevel, car, zipcode, hvalue, hyears, loan):
+    def _classification_function_zero(salary, commission, age, elevel, car, zipcode, hvalue, hyears, loan):
         """ classification_function_zero
 
         Parameters
         ----------
         age: float
-            First numeric attribute.
+            Numeric feature.
```

Comment:
Please include a proper description instead of only "Numeric/Categorical feature"
For example: `elevel` -> "Education Level" 

The same comment applies for the other classification functions